services/account/src/application.ts

```diff
@@ -0,0 +1,45 @@
+import {ApplicationConfig} from '@loopback/core';
+import {RestApplication} from '@loopback/rest';
+/* tslint:disable:no-unused-variable */
+import {
+  DataSourceConstructor,
+  juggler,
```

Comment:
Instead of doing this, would it not be better to do 
```js
import * as lbRepository from '@loopback/repository';
import * as lbBoot from '@loopback/boot';
```
and then the class would be defined as follows 
```js
export class AccountMicroservice extends lbBoot.BootMixin(lbRepository.RepositoryMixin(RestApplication)) {
```
That way we don't have to deal with the `unused-variable` ts error, and the IDE's auto formatting removing these unused imports?

Reply:
While that removes the `unused-variable` tslint error, it's good practice to import only what we need from those modules as it makes it clear what we use from them. This is the approach we take in `loopback-next` and it'd be good to follow it here.